### DISCUSSION TITLE
Great Resources For Leaning How To Work With Stable Diffusion

### DISCUSSION THREAD

**Discussion by JoshuaKimsey** [hlky/stable-diffusion#244] [General]
Wanted to share this here with everyone here, and tag @hlky and @altryne since this might be a good resource to add to the readme and the GUI markdown guide page too. A user on Reddit did a great guide titled "[How to get images that don't suck](https://www.reddit.com/r/StableDiffusion/comments/x41n87/how_to_get_images_that_dont_suck_a/?utm_source=share&utm_medium=web2x&context=3)." It goes over prompt building and tokenization, CFG scaling, sampler differences, seeds, and more. It would probably help a lot of newer users figure out some of the intricacies of getting better images from Stable Diffusion, and is definitely a good all-in-one source for those starting out. :)

**Comment by altryne** [comment]
@JoshuaKimsey would love it if you add to https://github.com/altryne/awesome-ai-art-image-synthesis/ ! 
That way you'll be a collaborator there as well (there's a tutorials section) 🙏

**Reply by JoshuaKimsey** [reply]
Will do!

**Reply by JoshuaKimsey** [reply]
Just did so, added a number of new sources for your resource guide! :)

**Reply by altryne** [reply]
And fixed my lamo typos! Thanks man! Much appreciated
